Libplanet/Blockchain/BlockChain.cs

```diff
@@ -507,18 +517,23 @@ bool render
                 HashDigest<SHA256>? tip =
                     Store.IndexBlockHash(Id.ToString(), -1);
 
-                ValidateNonce(block);
-
-                evaluations = block.Evaluate(
-                    currentTime,
-                    a => GetStates(new[] { a }, tip).GetValueOrDefault(a)
-                ).ToArray();
+                if (evaluateActions)
+                {
+                    ValidateNonce(block);
```

Comment:
It was intended, but I'm going to reconsider this.  Until yesterday, I thought tx nonces cannot be made because these requires the entire actions to be evaluated, but it turns out they probably does not really.

Reply:
👌

Reply:
I made tx nonces completely independent from action evaluation, and now these are not included in `RecentStates` message, but always calculated by each node instead, because these are merely adding numbers.  See also: de75e11d1935dffad78b9eb649672d3f6d4cefc5